integreat_compass/cms/urls/__init__.py

```diff
@@ -0,0 +1,14 @@
+"""
+Django URL dispatcher for the cms package.
+See :mod:`~integreat_compass.core.urls` for the other namespaces of this application.
+
+For more information on this file, see :doc:`django:topics/http/urls`.
+"""
+
+from django.urls import include, path
+
+#: The url patterns of this module (see :doc:`django:topics/http/urls`)
+urlpatterns = [
+    path("", include("integreat_compass.cms.urls.public")),
+    path("", include("integreat_compass.cms.urls.protected")),
+]
```

Comment:
```suggestion


#: The namespace for this URL config (see :attr:`django.urls.ResolverMatch.app_name`)
app_name = "cms"

#: The url patterns of this module (see :doc:`django:topics/http/urls`)
urlpatterns = [
    path("", include("integreat_compass.cms.urls.public", namespace="public")),
    path("", include("integreat_compass.cms.urls.protected", namespace="protected")),
]
```
I guess we could also make use of namespaces here and make the distinction based on it? But didn't try this out yet...
https://docs.djangoproject.com/en/4.2/topics/http/urls/#url-namespaces

Reply:
Sounds like a good idea to me!

Reply:
Small caveat seems to be that urls have to be referenced explicitly now, so `{% url "index" %}` no longer works, instead we have to write `{% url "cms:public:index" %}`. But at the same time this makes it more robust against mistakes and the access control looks far cleaner now, so I think it's worth it...

Reply:
Ah, true. But yes, not really a dealbreaker.